app-emacs/epl/epl-0.9.ebuild

```diff
@@ -0,0 +1,18 @@
+# Copyright 2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit elisp
+
+DESCRIPTION="A convenient high-level API for package.el"
+HOMEPAGE="https://github.com/cask/epl"
+SRC_URI="https://github.com/cask/epl/archive/refs/tags/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64"
+RESTRICT="test"
```

Comment:
Do you definitely need this? You should only need to add it if tests are broken or there's a dodgy target which means they run something obviously incorrect/they don't exist and the tests "fail" as a result